encoders/numeric/UMAPEncoder/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM jinaai/jina AS base
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN pip install -r requirements.txt
+
+# for testing the image
+FROM base
+RUN pip install pytest && pytest
+
+FROM base
+ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
```

Comment:
Maybe leave one more blank line at the end for each file.

Reply:
This file was auto-generated by the hub CLI wizard. Maybe the Dockerfile template for the wizard should be updated?

Reply:
`FROM base` is not needed before each command I believe. You can remove line 11 and 14